frontend/src/component/admin/auth/OidcAuth/OidcAuth.tsx

```diff
@@ -236,6 +241,34 @@ export const OidcAuth = () => {
                     data={data}
                     setValue={setValue}
                 />
+                <Grid container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Enable Additional Scopes</strong>
+                        <p>
+                            If you enable additional scopes Unleash will request
+                            the additional scopes needed to match configuration
+                            beyond openid, profile, and email. For example 'groups'.
```

Comment:
I wonder if this should be add groups scope instead, because now its only groups.

Reply:
Yeah I think you're right! I've changed it to be addGroupsScope and only be for the groups scope